scripts/ffmprovisr

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+if [[ $OSTYPE = darwin* ]] ; then
+    if [ -d /usr/local/Cellar/ffmprovisr ] ; then
+        ffmprovisr_path=$(find /usr/local/Cellar/ffmprovisr -iname 'index.html' | sort -M | tail -n1)
+    fi
```

Comment:
Hmm… I’m just thinking now if an `else` would not be better: either there is the folder, or the online version is used. Or do I miss something?